courses/models.py

```diff
@@ -540,6 +540,7 @@ class CourseRunEnrollment(EnrollmentModel):
         default=False,
         help_text="Indicates whether or not the request succeeded to enroll via the edX API",
     )
+    emails_subscription = models.BooleanField(default=True)
```

Comment:
It seems an overhead. First, the ideal situation is we will have the edX configured on RC or PROD but only it will be local setup. Secondly, the doesn't matter whether the edx enrolled or not. So whenever the course will be edx enrolled for the first, it will be considered True so as it comes by default subscribed for course emails.

Reply:
As long as we consider the ideal situation what you said makes sense but it would still be consistent when we look at the data overall. I'm okay with that but consider this scenario while keeping in mind:
1- We have `unenroll` option too, so consider a user unsubscribes from course emails and then unenrolls from the course. 
2- He reenrolls, now he is enrolled back in but his email subscriptions would still be off.(Should the be on when he/she reenrolls?)

If you are clear on the above scenario i'm okay with this change.

Reply:
Please create a separate ticket for that, we will encounter that scenario in a separate issue if it gets approved by the client.